src/routes/Home.jsx

```diff
@@ -29,7 +30,7 @@ const videogames = [
   },
 ];
 
-const Home = () => {
+const Home = ({ deleteButton, message }) => {
```

Comment:
Would it make sense not to add `message` as a prop? We could have `Home` get it directly from the store and this way we would be a step away from [prop drilling](https://dev.to/codeofrelevancy/what-is-prop-drilling-in-react-3kol#:~:text=Prop%20drilling%20occurs%20when%20a,difficult%20to%20manage%20and%20maintain.), which is usually undesired.

Reply:
Thanks for the feedback @dicodiaz! That's a great idea but the bad part of implementing it that way is that when you delete a videogame and then you go back home the `message` will be displayed in `Home component` and not only on `DeleteItem component`. Do you think it would be better to implement it that way?